src/main.ts

```diff
@@ -1,4 +1,11 @@
 import { createApp } from 'vue';
 import App from './App.vue';
 
-createApp( App ).mount( '#app' );
+interface Config {
+	rootSelector: string;
+}
+
+export default function createAndMount( config: Config ): void {
```

Comment:
I didn't notice it until now, but I think it might be more convenient to work with, if we return the mounted app as well.

Reply:
Done.

Reply:
Out of curiosity: what would we do with the mounted app on-wiki?